template/python3-flask/index.py

```diff
@@ -22,7 +23,13 @@ def fix_transfer_encoding():
 @app.route("/", defaults={"path": ""}, methods=["POST", "GET"])
 @app.route("/<path:path>", methods=["POST", "GET"])
 def main_route(path):
-    ret = handler.handle(request.get_data())
+    raw_body = os.getenv("RAW_BODY")
```

Comment:
For a semblance of consistency -> https://github.com/openfaas/templates/blob/master/template/node12/index.js#L11